securedrop/journalist.py

```diff
@@ -132,7 +132,7 @@ def login():
 
             flash(login_flashed_msg, "error")
         else:
-            app.logger.info("Successful login for '{}' with token {}".format(
+            app.logger.info("'{}' logged in with the token {}".format(
```

Comment:
This is lowercase l, but it replaces "Successful"

Reply:
I'm sorry but I don't understand what you mean ?

Reply:
Note that this string is never visible to the end user. It is stored in the server logs for analysis by the system administrator if needed.

Reply:
I mean it should be "Login" because it was "Successful" "S" for "L" whereas now it is "S" for "l"